index.js

```diff
@@ -1,19 +1,22 @@
 var parse = require('./parser').parse;
+var print = require('./printer').print;
 
 var cache = {};
 
-function stripLoc (doc) {
+function stripLoc (doc, removeRoot) {
```

Comment:
I disagree, I think it's the first argument that is poorly named. It should be `ast`. And the second argument is "are we removing the root of the *current ast we are recursing on*".